docs/control-structures.rst

```diff
@@ -273,18 +273,17 @@ i.e. the following is not valid: ``(x, uint y) = (1, 2);``
 Complications for Arrays and Structs
 ------------------------------------
 
-The semantics of assignments are a bit more complicated for
-non-value types like arrays and structs.
-Assigning *to* a state variable always creates an independent
-copy. On the other hand, assigning to a local variable creates
-an independent copy only for elementary types, i.e. static
-types that fit into 32 bytes. If structs or arrays (including
-``bytes`` and ``string``) are assigned from a state variable
-to a local variable, the local variable holds a reference to
-the original state variable. A second assignment to the local
-variable does not modify the state but only changes the
-reference. Assignments to members (or elements) of the local
-variable *do* change the state.
+The semantics of assignments are more complicated for non-value types like arrays and structs.
+
+Assigning **to a state variable** always creates an independent copy.
+
+Assigning **to a local variable** with the same data location creates an independent copy only for
+elementary types, i.e. static types that fit into 32 bytes.
+
+If assigning structs or arrays (including ``bytes`` and ``string``) **from a state variable to a  local variable**,
+the local variable holds a reference to the original state variable, **no copy is made**.
+A second assignment to the local variable does not modify the state but only changes the
+reference. Assignments to members (or elements) of the local variable **do** change the state.
```

Comment:
I know this text was already like this before, but it doesn't sound very precise to me.
Assigning from a state array to a local memory array does create a copy, and I don't see that in the text.
Also, assigning a local reference to a state variable does modify the state, via the reference.

Reply:
@leonardoalt For

> Also, assigning a local reference to a state variable does modify the state, via the reference.

Which part of the text above should that apply to, or is it something seperate and new?